src/test/java/uk/gov/companieshouse/filinghistory/consumer/kafka/ConsumerPositiveComprehensiveIT.java

```diff
@@ -67,6 +67,7 @@ public void setup() {
 
     @ParameterizedTest
     @CsvSource({
+            "363s/363s",
```

Comment:
should be in the annual_return directory, using the mongo data.category field as the naming convention